package.json

```diff
@@ -2489,6 +2489,316 @@
+									{
+										"key": "languageId:latex",
+										"name": "LaTeX",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format region or buffer",
+														"type": "command",
+														"command": "editor.action.format"
+													},
+													{
+														"key": "b",
+														"name": "Format buffer",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "c",
+														"name": "Format changes",
+														"type": "command",
+														"command": "editor.action.formatChanges"
+													},
+													{
+														"key": "s",
+														"name": "Format selection",
+														"type": "command",
+														"command": "editor.action.formatSelection"
+													}
+												]
+											},
+											{
+												"key": "b",
+												"name": "+Backend",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "l",
+														"name": "View Workshop Messages",
+														"type": "command",
+														"command": "latex-workshop.log"
+													},
+													{
+														"key": "m",
+														"name": "Insert root magic comment",
+														"type": "command",
+														"command": "latex-workshop.addtexroot"
+													}
+												]
+											},
+											{
+												"key": "c",
+												"name": "+Build",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "c",
+														"name": "Build Project",
+														"type": "command",
+														"command": "latex-workshop.build"
+													},
+													{
+														"key": "i",
+														"name": "Show compilation info",
+														"type": "command",
+														"command": "latex-workshop.showCompilationPanel"
+													},
+													{
+														"key": "k",
+														"name": "Kill compiler process",
+														"type": "command",
+														"command": "latex-workshop.kill"
+													},
+													{
+														"key": "l",
+														"name": "Clean up auxiliary files",
+														"type": "command",
+														"command": "latex-workshop.clean"
+													},
+													{
+														"key": "l",
+														"name": "View compiler logs",
+														"type": "command",
+														"command": "latex-workshop.compilerlog"
+													},
+													{
+														"key": "r",
+														"name": "Build with recipe",
+														"type": "command",
+														"command": "latex-workshop.recipes"
+													}
+												]
+											},
+											{
+												"key": "f",
+												"name": "+Fonts",
```

Comment:
Should this be +Text and `x`?

Reply:
Spacemacs has the equivalent under too much nesting IMO (ie. `m x f *`). I wanted to bring it up one level but we could put it under the same `m x f *` if you prefer.

Reply:
I agree, I am wondering if we can flatten it by removing the fonts, so we will have direct access to `m x b` for bold for example.

Reply:
Agreed. I flattened it by removing the fonts as you suggested.